lib/membrane_caps_audio_raw.ex

```diff
@@ -283,6 +283,17 @@ defmodule Membrane.Caps.Audio.Raw do
     end
   end
 
+  @doc """
+  Returns a binary which corresponds to the silence during the given interval
+  of time in given caps' fromat
```

Comment:
Maybe adding a unit test for this function would be an overkill, but some example (working as doctest) is welcome

Reply:
For reference see https://hexdocs.pm/ex_unit/ExUnit.DocTest.html and https://github.com/membraneframework/bunch/blob/v0.1.1/lib/bunch/enum.ex#L262